pentaho-requirejs-utils/pom.xml

```diff
@@ -50,6 +50,12 @@
       <version>${junit.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
@pamval This dependency was already in use in another bundle (https://github.com/pentaho/pentaho-osgi-bundles/blob/759a91795da494408c3827c45102aefc7bc9657d/pentaho-webjars-deployer/pom.xml) so I assumed that it's a previously approved one.